packages/story-editor/src/components/publishModal/content/index.js

```diff
@@ -52,7 +53,9 @@ const _MainStoryInfo = styled.div`
   display: flex;
   flex-direction: column;
   padding: 0 16px;
-  overflow: scroll;
+  overflow-y: scroll;
```

Comment:
We currently don't have enough content in here to warrant it scrolling, but I am not sure what v2 will look like, since it doesn't hurt to have it here I left it. But can definitely just remove all together if y'all prefer.